docs/en/setup/backend/backend-alarm.md

```diff
@@ -51,13 +54,22 @@ rules:
     op: <
     period: 10
     count: 4
+  service_resp_time_percentile_rule:
+    # Metrics value need to be long, double or int
+    metrics-name: service_percentile
+    op: ">"
+    threshold: 1000,1000,1000,1000,1000
```

Comment:
I don't know how other people think of this, I'm confused with the order of the thresholds at first glance, `p99` -> `p50` or `p50` -> `p99`, and seems only [1] tells the order, vaguely though

Reply:
Do you want me to put comments here? Or put doc at some places?

Reply:
> Do you want me to put comments here? Or put doc at some places?

I prefer to point out the order here, users don't need to refer to other doc when they're uncertain with the order 😄